docker-compose.yml

```diff
@@ -1,9 +1,9 @@
-# Wazuh App Copyright (C) 2019 Wazuh Inc. (License GPLv2)
+# Wazuh App Copyright (C) 2020 Wazuh Inc. (License GPLv2)
 version: '2'
 
 services:
   wazuh:
-    image: wazuh/wazuh:3.11.4_7.6.1
+    image: wazuh/wazuh:3.12.4_7.6.1
```

Comment:
```suggestion
    image: wazuh/wazuh:3.12.0_7.6.1
```

Reply:
Fixed for all tags in https://github.com/wazuh/wazuh-docker/pull/316/commits/cb60fc8b771a04b2c3beb652c305ffd5b316687e

Greetings !